docs/data/ampli/plugin.md

```diff
@@ -2,38 +2,209 @@
 title: Plugin
 description: Use plugins to extend Amplitude by running a sequence of custom code on every event. This pattern is flexible and you can use it to support event enrichment, transformation, filtering, routing to third-party destinations, and more.
 template: guide.html
-
 ---
 
 !!!note
-    Plugins are supported in the latest version of Ampli when used in conjunction with the [new Amplitude browser SDKs](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/migration/). If you are using an older version of Ampli or a [Maintenance Amplitude SDK](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/migration/), see **[Middleware](../middleware/)**.
+Plugins are supported in the latest version of Ampli when used in conjunction with the [new Amplitude browser SDKs](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/migration/). If you are using an older version of Ampli or a [Maintenance Amplitude SDK](https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/migration/), see **[Middleware](../middleware/)**.
```

Comment:
Please revert this change. This is the syntax for [admonitioins](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage). And also it helps to do a [self review ](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/wiki#3-run-a-local-server-and-preview-your-changes)of all chnages.

Reply:
+1